packages/edit-site/src/components/global-styles/utils.js

```diff
@@ -120,8 +120,7 @@ function findInPresetsBy(
 			for ( const origin of origins ) {
 				const presets = presetByOrigin[ origin ];
 				if ( presets ) {
-					const presetObject = find(
-						presets,
+					const presetObject = presets.find(
```

Comment:
Presets for a particular origin are expected to be arrays: 

https://github.com/WordPress/gutenberg/blob/c703ff046dee5056fb558f09bccf6b24505a2f03/docs/how-to-guides/themes/theme-json.md